executor/aggregate.go

```diff
@@ -395,32 +395,48 @@ func (w *HashAggPartialWorker) shuffleIntermData(sc *stmtctx.StatementContext, f
 }
 
 // getGroupKey evaluates the group items and args of aggregate functions.
-func (w *HashAggPartialWorker) getGroupKey(sc *stmtctx.StatementContext, row chunk.Row) ([]byte, error) {
-	w.groupValDatums = w.groupValDatums[:0]
+func (w *HashAggPartialWorker) getGroupKey(sc *stmtctx.StatementContext, input *chunk.Chunk) (err error) {
```

Comment:
The code in this function is almost the same as the HashAggExec.getGroupKey. Can we merge them?